roles/ci_dcn_site/tasks/pre-ceph.yml

```diff
@@ -0,0 +1,90 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Render the pre-ceph NodeSet values.yaml
+  vars:
+    _edpm_instance_dict: "{{ cifmw_networking_env_definition.instances }}"
+    _edpm_network_dict: "{{ cifmw_networking_env_definition.networks }}"
+    _ssh_authorizedkeys: "{{ lookup('file', '~/.ssh/id_cifw.pub', rstrip=False) }}"
+    _ssh_private_key: "{{ lookup('file', '~/.ssh/id_cifw', rstrip=False) }}"
+    _ssh_public_key: "{{ lookup('file', '~/.ssh/id_cifw.pub', rstrip=False) }}"
+    _migration_priv_key: "{{ lookup('file', '~/ci-framework-data/artifacts/nova_migration_key', rstrip=False) }}"
+    _migration_pub_key: "{{ lookup('file', '~/ci-framework-data/artifacts/nova_migration_key.pub', rstrip=False) }}"
+  ansible.builtin.template:
```

Comment:
No one needs to update the CRs generated by the playbook. They are used only for the CI test.

Reply:
I'm not that sure, cause we always said that nothing should change and that the very end some tweaks are always needed, we have plenty of examples in DS jobs:

- Internal CA (in fully virtual reproducer deployments the reproducer injects the CA underneath, a detail that is usually missed)
- Repositories setup: (in fully virtual reproducer deployments the reproducer injects the repositories, a detail that is usually missed)
- EDPM images replacement, ie: the telemetry exporter. If overridden (done in every single DS job we have) podman credentials needs to be updated too.

I'm not going to stop this PR because of this point, but keep in mind that the job the team will build based on this content will be complicated to tweak if required. 
PS: Details, like the internal NTP server are stuff we usually hide upstream and we kustomize with the needed value in DS jobs.